schemas/groups/resources.json

```diff
@@ -220,6 +220,22 @@
               "type": "string",
               "description": "Location of the note"
             },
+            "group": {
+                "type": "string",
+                "description": "Name of a group or collection to which the note is attached"
+            }, 
+            "author": {
+                "type": "string",
+                "description": "Name of the creator of the note"
+            }, 
+            "readOnly": {
+                "type": "boolean",
+                "description": "Indicates whether the whether the content of the Note is permitted to be changed."
```

Comment:
double whether

Reply:
A more future proof way to implement this would be to add an optional enum property `accessType` with one value, `READONLY`. This way it is easier to extend.

Reply:
Keep in mind that read/write/delete can only be controlled on the local server. Once a note (or any signalk object) is exported to another client/server you can no longer control editing. 

If the server provides per-key security the note can have permissions set the same as you would a file in linux, eg -rw-r---r--- (644) which achieves a more flexible result and is standard across all keys. Likewise -rw-r------ (640) would stop other users seeing the note unless in the proper group.

This can be communicated to the client via `meta._attr`, however any disallowed attempt to view/save the note would fail anyway. 

So I dont think we even need `accessType`

Reply:
If the client can get read/write status via a means like `meta._attr` then this would work. 
Is this a documented approach for developers to follow?

Reply:
The 'real' privileges for something like a note entity would be CRUD, not Linux permissions, so I favour the enum solution. However I can live with anything that allows the client to get read/write status from the server, preferably using a documented approach.

Reply:
Linux permissions basically provide CRUD per owner/group/others. See below - I am trying to avoid adding one-off solutions so that server implementations and security can be applied easily across all current and future keys.
The UNIX/Linux solution isnt perfect but for something that was invented 40 years ago, and is still viable in the internet age suggests its a pretty good design.

Reply:
As per comments below, permissions should have a separate PR.